content/en/docs/concepts/security/api-server-bypass-risks.md

```diff
@@ -0,0 +1,147 @@
+serves as authorization to interact with the kubelet API. The exact access will depend on
+which sub-resource access has been granted, as detailed in [kubelet authorization](https://kubernetes.io/docs/reference/access-authn-authz/kubelet-authn-authz/#kubelet-authorization).
+
+Direct access to the kubelet API is not subject to admission control and is not logged
+by Kubernetes audit logging. As such, an attacker who can use this API may be able to
+controls otherwise intended to detect/prevent certain actions.
+
+The kubelet API can be configured to authenticate requests in a number of ways. 
+By default the kubelet configuration is to allow anonymous access. Most Kubernetes providers
+set configuration to use webhook and certificate authentication. This would enable the
+control plane to ensure that the caller is authorized to access the `nodes` API or its
+subresource. If set to anonymous access, there is no such assertion. 
+
+
+
+### Mitigations
+
+- Minimize access to subresources of the `nodes` API, using mechanisms such as RBAC.
+  This access is likely to be needed for some services (for example: monitoring services)
+  , but should not be provided where not needed.
+- Consider restricting access to the kubelet port, to only allow access from specified
+  and trusted IP address ranges.
+- Ensure that kubelet authentication is set to webhook or certificate mode. 
+
+## Etcd API
+
+Kubernetes clusters use etcd as a datastore. The etcd service listens on TCP port 2379;
+the only clients that need access are the Kubernetes API server and any backup tooling
+that you use. Direct access to this API allows for disclosure or modification of any
+data held in the cluster.
+
+Access to the etcd API is typically managed by client certificate authentication.
+Any certificate issued by a certificate authority that etcd trusts will allow full access
+to the data stored inside etcd.
+
+Direct access to etcd is not subject to Kubernetes admission control and is not logged
+by Kubernetes audit logging. An attacker who is able to gain read access to the API server's
+etcd client certificate (or is able create a new trusted client certificate) can gain
+cluster admin rights by accessing cluster secrets or modifying access rules. Even without
+elevating their privileges, an attacker who can modify etcd can retrieve any API object
+or create new workloads inside the cluster.
+
+You can configure etcd in a number of ways. For the most part Kubernetes providers configure 
+etcd to use mutual TLS (both client and server verify each other's certificate for authentication).
+There is not a widely accepted implementation of authorization for the etcd API, although
+the feature does exist. Since there is no authorization model, this means that any certificate
+with client access can be used to gain full access to etcd. Typically, client certificates
+that are only used for health checksing can also grant full read and write access.
```

Comment:
```suggestion
There is no widely accepted implementation of authorization for the etcd API, although
the feature exists. Since there is no authorization model, any certificate
with client access can be used to gain full access to etcd. Typically, client certificates
that are only used for health checking can also grant full read and write access.
```

Line "the feature exists" to the feature